supply-chain/config.toml

```diff
@@ -320,7 +320,7 @@ version = "0.10.0"
 criteria = "safe-to-deploy"
 
 [[exemptions.h2]]
-version = "0.3.16"
+version = "0.3.19"
```

Comment:
Could this diff be audited to avoid adding a new exemption? (assuming that this diff is a relatively small one hopefully)

Reply:
I could, I just figured that if we have an exemption I'd keep bumping it. I've been thinking about our exemptions of the tokio/hyper family (which this is a part of) is a wildcard audit for carl and sean.

Reply:
Ok sounds reasonable 👍